core/src/test/java/io/confluent/kafka/schemaregistry/SSLClusterTestHarness.java

```diff
@@ -62,14 +64,10 @@ protected KafkaConfig getKafkaConfig(int brokerId) {
       throw new RuntimeException(e);
     }
 
-    props.setProperty("auto.create.topics.enable", "true");
-    props.setProperty("num.partitions", "1");
+    injectProperties(props);
     if (requireSSLClientAuth()) {
       props.setProperty("ssl.client.auth", "required");
     }
-    // We *must* override this to use the port we allocated (Kafka currently allocates one port
-    // that it always uses for ZK
-    props.setProperty("zookeeper.connect", this.zkConnect);
```

Comment:
I think this was legacy code that is no longer applicable. Tests pass with this line removed, so I removed it for cleanliness reasons. I'm happy to add it back to be safe.